[OSDOCS-6942]:OSD on GCP can be installed into Shared VPC (XPN) (#66312)
openshift/openshift-docs: modules/deleting-cluster.adoc

. Select *Delete cluster* from the *Actions* drop-down menu.

. Type the name of the cluster highlighted in bold, then click *Delete*. Cluster deletion occurs automatically.

+

> 

Reviewer: I don't think you need the extra line here.

[NOTE]
====

> If you delete a cluster that was installed into a GCP shared VPC, you must inform the VPC owner of the host project to remove the IAM policy roles granted to the service account that was referenced during the creation of the cluster.

Reviewer: ```suggestion
If you delete a cluster that was installed into a GCP shared VPC, inform the Shared VPC Admin to remove the IAM policy roles granted to the service account that was referenced during cluster creation.
```
We use "VPC Owner" as a proper noun on the most recently updated Shared VPC page in ROSA (https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-shared-vpc-config.html), but it looks like Google uses "Shared VPC Admin" (https://cloud.google.com/vpc/docs/provisioning-shared-vpc#set-up-shared-vpc), hence the name change.
